test/11_LairotcafTest.js

```diff
@@ -0,0 +1,23 @@
+var expect    = require("chai").expect;
+var lairotcaf = require("../scripts/11_Lairotcaf");
+
+describe("reverse factorial test", function() {
+  it("should return the string 6!", function() {
+    var moddedString  = lairotcaf.reverseFactorial( 720 );
+
+    expect(moddedString).to.equal( '6!' );
+  });
+
+  it("should return the string none", function() {
```

Comment:
include why in test like `"Returns the string none if input has no factorial base"` or something more concise